config/initializers/devise.rb

```diff
@@ -247,9 +247,9 @@
     }
 
     config.omniauth :github,
-      Errbit::Config.github_client_id,
-      Errbit::Config.github_secret,
-      github_options
+                    Errbit::Config.github_client_id,
+                    Errbit::Config.github_secret,
+                    github_options
```

Comment:
This irks me, but I could probably learn to live with it. In general, I really prefer multi-line indentation to be the normal, 2-space style. I tend to make use of namespaces as well as descriptive variable names and its really easy to end up with super long lines. It hasn't happened in this case thankfully, but its very easy to end up there.